packages/orchestrator-ui-components/src/components/WfoSummary/WfoSummaryCardListItem.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, ReactNode, useState } from 'react';
+
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiTextColor } from '@elastic/eui';
+
+import { WfoOptionalLink } from '@/components/WfoOptionalLink';
+
+export type SummaryCardListItem = {
+    title: string;
+    value: ReactNode;
+    url?: string;
+};
+
+export type WfoSummaryCardListItemProps = {
+    title: string;
+    value: ReactNode;
+    url?: string;
+};
+
+export const WfoSummaryCardListItem: FC<WfoSummaryCardListItemProps> = ({
+    title,
+    value,
+    url,
+}) => {
+    const [hoverState, setHoverState] = useState(false);
+
+    return (
+        <WfoOptionalLink href={url}>
+            <EuiFlexGroup
+                style={{ paddingBlock: 10 }}
+                onMouseOver={() => setHoverState(true)}
+                onMouseLeave={() => setHoverState(false)}
+                gutterSize="none"
+            >
+                <EuiFlexItem>
+                    <EuiTextColor
+                        color={url ? '#397dc2' : 'black'}
```

Comment:
Theme values?

Reply:
using the link color form the theme. Not the same but looks the same